applications/crossbar/src/modules/cb_port_requests.erl

```diff
@@ -425,14 +449,32 @@ date_as_configured_timezone(Date, Time, FromTimezone) ->
 
 -spec post(cb_context:context(), path_token()) -> cb_context:context().
 post(Context, Id) ->
+    maybe_post_save(Context, Id, has_new_comments(Context)).
+
+-spec maybe_post_save(cb_context:context(), kz_term:ne_binary(), kz_json:objects()) -> cb_context:context().
+maybe_post_save(Context, _, []) ->
     Context1 = save(Context),
     case cb_context:resp_status(Context1) of
         'success' ->
-            _ = maybe_send_port_comment_notification(Context1, Id),
             Doc = cb_context:doc(Context1),
             cb_context:set_resp_data(Context1, knm_port_request:public_fields(Doc));
```

Comment:
Can you call `filter_private_comments/2` just like `save_then_maybe_notify/3` in here too?